test/files/run/repl-trim-stack-trace.check

```diff
@@ -1,28 +1,28 @@
-Welcome to Scala
+Welcome to Scala %1$#s (%3$s, Java %2$s).
```

Comment:
could we just remove the `Welcoming` trait from repl-trim-stack-trace and t7805-repl-i, or are we testing something useful here?